doc/source/getting_started/install.rst

```diff
@@ -201,6 +201,22 @@ Installing from source
 
 See the :ref:`contributing guide <contributing>` for complete instructions on building from the git source tree. Further, see :ref:`creating a development environment <contributing_environment>` if you wish to create a pandas development environment.
 
+Installing the development version of pandas
+~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This needs to match the length of the title

Reply:
Do I need to compile the [bug report form](https://github.com/pandas-dev/pandas/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml)?
Isn't that information already specified in the related issue?

Reply:
@lpizzinidev I think the request is to add a link to this in the bug report form, so that in the future, when people report bugs, it's easier for them to check whether the bug they're reporting is still present on the `main` branch